src/Features/Core/Portable/Diagnostics/DocumentAnalysisExecutor.cs

```diff
@@ -109,55 +113,56 @@ public async Task<IEnumerable<DiagnosticData>> ComputeDiagnosticsAsync(Diagnosti
                 }
             }
 
-            var skippedAnalyzerInfo = document.Project.GetSkippedAnalyzersInfo(_analyzerInfoCache);
```

Comment:
Lot of code in this file has now been simplified as all this happens now at a centralized place.